pretix_sepadebit/signals.py

```diff
@@ -59,6 +68,70 @@ def register_csv(sender, **kwargs):
     return DebitList
 
 
+@receiver(register_mail_placeholders, dispatch_uid="payment_sepadebit_placeholders")
+def register_mail_renderers(sender, **kwargs):
+
+    ph = [SimpleFunctionalMailTextPlaceholder(
+          'due_date', ['sepadebit_payment'], lambda sepadebit_payment: sepadebit_payment.due.date, sample=date.today()
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'account', ['sepadebit_payment'], lambda sepadebit_payment: sepadebit_payment.info_data.get('account', " "), sample="Max Mustermann"
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'iban', ['sepadebit_payment'], lambda sepadebit_payment: sepadebit_payment.info_data.get('iban')[0:4] + " **** " + sepadebit_payment.info_data.get('iban')[-4:], sample="DE02 **** 2051"
```

Comment:
I'd drop the spaces. Also, we should test that this doesn't cause any issues with the markdown rendering in emails and causes things to be printed as bold :D Maybe 1234xxxx5689 is safer?